typescript/index.d.ts

```diff
@@ -47,7 +47,7 @@ export interface AtomOptions<T> {
 export function atom<T>(options: AtomOptions<T>): RecoilState<T>;
 
 // selector.d.ts
-type GetRecoilValue = <T>(recoilVal: RecoilValue<T>) => T;
+export type GetRecoilValue = <T>(recoilVal: RecoilValue<T>) => T;
 
 type SetRecoilState = <T>(
```

Comment:
Please also export `SetRecoilState` and `ResetRecoilState`

Reply:
thank you, added in f6c8227